src/vdirectorytree.cpp

```diff
@@ -35,10 +35,10 @@ VDirectoryTree::VDirectoryTree(QWidget *parent)
 
     initShortcuts();
 
-    connect(this, SIGNAL(itemExpanded(QTreeWidgetItem*)),
-            this, SLOT(handleItemExpanded(QTreeWidgetItem*)));
-    connect(this, SIGNAL(itemCollapsed(QTreeWidgetItem*)),
-            this, SLOT(handleItemCollapsed(QTreeWidgetItem*)));
+    connect(this, SIGNAL(itemExpanded(QTreeWidgetItem *)),
```

Comment:
It looks like that a format tool is applied for this file? Please revert it to reduce irrelevant changes.

Reply:
Thank you for your advice, I have corrected my mistake.